src/System.Private.ServiceModel/tools/scripts/RemoveFirewallPorts.cmd

```diff
@@ -0,0 +1,39 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+echo Start Remove all ports WCF test services use
+
+SET __EXITCODE=0
+
+net session > NUL
+if NOT [%ERRORLEVEL%]==[0] ( (
```

Comment:
There are too many open parentheses here.  Unless I manually remove this one open paren, running this command yields:
```
c:\OSS\wcf\src\System.Private.ServiceModel\tools\scripts>RemoveFirewallPorts.cmd
Start Remove all ports WCF test services use
The syntax of the command is incorrect.
```